lib/page/home/widget/home_surveys_item_widget.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import 'package:survey/gen/assets.gen.dart';
+import 'package:survey/gen/colors.gen.dart';
+import 'package:survey/model/survey_model.dart';
+import 'package:survey/resource/dimens.dart';
+
+class HomeSurveysItemWidget extends StatelessWidget {
+  final SurveyModel survey;
+  final VoidCallback onNextButtonPressed;
+
+  const HomeSurveysItemWidget({
+    Key? key,
+    required this.survey,
+    required this.onNextButtonPressed,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        image: DecorationImage(
+          image: Image.network(survey.coverImageUrl).image,
+          colorFilter: ColorFilter.mode(
+            Colors.black.withOpacity(0.5),
+            BlendMode.darken,
+          ),
+          fit: BoxFit.cover,
+        ),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.all(Dimens.space20),
+        child: Column(
+          children: [
+            Expanded(
+              child: const SizedBox.shrink(),
+            ),
+            Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: [
```

Comment:
Why do we need to use 2 Column widgets here? Not sure what's your purpose here.

Can we optimize to use only 1?

Reply:
Nice catch, updated 🚀